js/node_shared/unix.js

```diff
@@ -237,8 +237,13 @@ function unix_getcwd(vunit) {
 //Provides: unix_chdir
 //Requires: caml_sys_chdir
 function unix_chdir(path) {
+  /* When we do a Unix chdir, this signals that we would like for the Node
```

Comment:
I don't really see the value of this comment ... yes when you call chdir you want the process to change directory.

Reply:
The distinction here is that in Js_of_ocaml code, since the Node process is only simulating an OCaml one, it keeps a field with the OCaml simulation's current directory, and changes that upon a `Sys.chdir`, and not the actual Node process's path. I was making the distinction that a `Unix.chdir` will change the Node directory also.